packages/mdc-segmented-button/segment/_segment-theme.scss

```diff
@@ -0,0 +1,157 @@
+//
+// Copyright 2020 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+// Theme: Variables, functions, and mixins required to theme component segment
+
+// TODO: is it ok to use mdc-button?
+@use "@material/button";
+@use "@material/theme/variables" as theme-variables;
+@use "@material/feature-targeting/functions" as feature-targeting-functions;
+@use "@material/feature-targeting/mixins" as feature-targeting-mixins;
+@use "@material/typography/mixins" as typography-mixins;
+
+$unselected-ink-color: rgba(theme-variables.prop-value(on-surface), 0.6);
```

Comment:
@asyncLiz Should we start using `custom-properties.create()` API for colors that apply custom alpha from now on?

```scss
@use '@material/theme/custom-properties';

custom-properties.create(--mdc-segmented-button-unselected-text-color, rgba(on-surface, 0.6))
```

Reply:
As per the discussion offline with Liz we should use `custom-properties.create()` API to use custom properties.

Reply:
I don't understand how the custom-properties works. Is it used something like `$unselected-ink-color: custom-properties.create(--mdc-segmented-button-unselected-ink-color, rgba(on-surface, 0.6));`? Or does it need to be used inside of an `@include` statement?

Reply:
`custom-properties.create()` & `custom-properties.apply()` APIs works like a polyfill for CSS custom properties to support IE11 browser.

Example:

```scss
$color: custom-properties.create(--test, red);

.test {
  @include custom-properties.apply('background-color', $color);
}
```

emits following CSS:

```css
.test {
  background-color: red;
  background-color: var(--test, red);
}
```

Your use case looks something like this:

```scss
@use '@material/theme/custom-properties';

$unselected-text-color: custom-properties.create(--mdc-segmented-button-unselected-text-color, rgba(on-surface, 0.6))

.mdc-segmented-button__foo-bar {
  @include custom-properties.apply('color', $unselected-text-color);
}
```